src-tauri/src/platform_specific/linux.rs

```diff
@@ -1,8 +1,78 @@
 // Linux specific code
 
 use regex::Regex;
+use std::env;
 use std::process::Command;
 
+fn get_proton_dir() -> Option<String> {
+    let steam_dir = steamlocate::SteamDir::locate()?;
+    let compat_dir = format!("{}/compatibilitytools.d/", steam_dir.path.display());
+
+    Some(compat_dir)
+}
+
+/// Downloads and installs NS proton
+/// Assumes Steam install
+pub fn install_ns_proton() -> Result<(), thermite::prelude::ThermiteError> {
+    // Get latest NorthstarProton release
```

Comment:
Being in a location with terrible internet rn I would prefer if there were `log::info!()`s  before fetching newest NS proton release version and downloading  corresponding zip. At least when testing rn, I was just looking at an essentially blank terminal waiting for something to happen ^^

Reply:
going further, this should probably be made async (I think they are called tasks in tauri?) so the UI keeps rendering while its happening.

Reply:
Oh yeah, good call. Though it should be enough to just make the wrapper functions async ^^